crates/full-node/sov-sequencer/src/preferred/rate_limiter/limiter.rs

```diff
@@ -272,12 +274,18 @@ impl<K: Hash + Eq + Debug + Send + Sync + 'static, S: Spec> RateLimiter<K, S> {
         let entry_count = self.data.entry_count();
         let limiter_type = self.limiter_type;
 
-        sov_metrics::track_metrics(|tracker| {
-            tracker.submit(RateLimiterMetrics {
-                value: entry_count,
-                limiter_type,
+        if self.metric_counter % 100 == 0 {
```

Comment:
Can we make optional config param, so in case we can change it without recompilation and other stuff?